partials/supported-libraries.adoc

```diff
@@ -0,0 +1,21 @@
+.Supported Libraries for {PRODUCT} v{VERSION}
+[cols="1,2"]
+|===
+| Library | Supported Version
+
+| `Quarkus`
+| `2.16.7.Final`
+
+| `Kogito Runtimes`
+| `1.40.0.Final`
+
+| `Kogito AddOns`
+| `1.40.0.Final`
+
+| `Drools`
+| `8.40.0.Final`
+
+| `KIE Tools`
+| `0.30.0`
```

Comment:
maybe the versions could be some variables form ascidoc point of view

Reply:
Interesting, this isn't being added anywhere, either.